extra/migrations/20160411000000_migrate_libvirt_settings.rb

```diff
@@ -0,0 +1,29 @@
+require 'fileutils'
+require 'yaml'
+
+class MigrateVirshToLibvirtConfig < ::Proxy::Migration
+  def migrate
+    input_yaml = YAML.load_file(File.join(src_dir, 'settings.yml'))
+    copy_original_configuration_except(path("dhcp_virsh.yaml", "dns_virsh.yml"))
```

Comment:
This doesn't look right - ```path``` merges individual components into a filepath: https://github.com/theforeman/smart-proxy/blob/develop/extra/migrate_settings.rb#L72. You should use it on the line above though.